unyt/array.py

```diff
@@ -1178,10 +1151,6 @@ def to_astropy(self, **kwargs):
         >>> data.to_astropy()
         <Quantity [3., 4., 5.] g / cm3>
         """
-        if _astropy.units is None:
```

Comment:
What happens if AstroPy isn't installed?

Reply:
The error message below actually never gets printed because doing `_astropy.units` raises an `ImportError` from the on-demand import machinery. I could have made this check do `if isinstance(_astropy, NotAModule)` instead, but decided against that because the error message from the on-demand import machinery is perfectly good.